app/src/main/res/layout/sheet_bp_passport.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  android:padding="@dimen/spacing_16">
+
+  <TextView
+    android:id="@+id/bpPassportNumberTextview"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="@dimen/spacing_16"
+    android:layout_marginBottom="@dimen/spacing_16"
+    android:gravity="center"
+    android:textAppearance="@style/Clinic.V2.TextAppearance.Black"
+    app:layout_constraintBottom_toTopOf="@+id/registerNewPatientButton"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toTopOf="parent"
+    tools:text="BP Passport 1111111" />
+
+  <com.google.android.material.button.MaterialButton
+    android:id="@+id/registerNewPatientButton"
+    style="?attr/materialButtonStyle"
+    android:layout_width="@dimen/spacing_0"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="@dimen/spacing_16"
+    android:layout_marginBottom="@dimen/spacing_16"
+    android:text="@string/sheet_bp_passport_register_new_patient"
+    android:textAllCaps="true"
```

Comment:
Can we use the text appearance here, please? It will be `Clinic.V2.TextAppearance.Button1`